### PR TITLE
Improve note by linking to the final commit

### PR DESCRIPTION
**Pull request by mehmetoguzderin**
Reduces influence of directory changes in being able to track history for what's meant by final update through direct link of commit

### CODE REVIEW

**Comment by mehmetoguzderin** [comment]
@furkanakkurt1335 please merge as neither self-approve nor bypass is available. I'd like this change to be reflected in document as the corresponding contributor.

File: Translations/otk-orkh-202311.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 :toc:
 
-NOTE: This document, titled "UDTW23 Translated Sentences for Old Turkish language (otk) and Old Turkic script (orkh)," is designated with version "202311" to be partial and will no longer receive updates. Any future work or modifications related to the subject matter of this document, particularly authoring for the language and script pair of Old Turkish and Old Turkic, should be pursued by creating a separate document. This approach ensures the content remains consistent and historically accurate as of its last update.
+NOTE: This document, titled "UDTW23 Translated Sentences for Old Turkish language (otk) and Old Turkic script (orkh)," is designated with version "202311" to be partial and will no longer receive updates. Any future work or modifications related to the subject matter of this document, particularly authoring for the language and script pair of Old Turkish and Old Turkic, should be pursued by creating a separate document. This approach ensures the content remains consistent and historically accurate as of its last and final update in the link:https://github.com/ud-turkic/udtw23/commit/f449e40e423015d687e69e6bbf421517cff65fb0++[f449e40] commit.
```
Comment (furkanakkurt1335) [inline Translations/otk-orkh-202311.adoc:4]: Can you make the link in markdown format?

**Review by furkanakkurt1335 — CHANGES_REQUESTED** [review]
Thank you for the update.

File: Translations/otk-orkh-202311.adoc (hunk 1 above)
Comment (mehmetoguzderin) [reply Translations/otk-orkh-202311.adoc:4]: @furkanakkurt1335 The document is in Asciidoc format, hence, this renders correctly

Comment (furkanakkurt1335) [reply Translations/otk-orkh-202311.adoc:4]: ah, sorry. Merging.

Comment (mehmetoguzderin) [reply Translations/otk-orkh-202311.adoc:4]: @furkanakkurt1335 TYVM!

**Pull request merged**